src/ui/public/query_bar/directive/__tests__/query_bar.js

```diff
@@ -130,19 +130,5 @@ describe('queryBar directive', function () {
 
   });
 
-  describe('typeahead key', function () {
-
-    it('should use a unique typeahead key for each appName/language combo', function () {
```

Comment:
I think we need to keep this test and update it to make sure the correct PersistedLog is being used when the language changes. Right now it seems to be broken, if I switch languages the local storage key does not change.